pages/linux/flashrom.md

```diff
@@ -0,0 +1,24 @@
+# flashrom
+
+> Read, write, verify and erase flash chips.
+> More information: <https://man.archlinux.org/man/flashrom.8>.
```

Comment:
```suggestion
> More information: <https://manned.org/flashrom>.
```
Since it isn't an Arch specific command, we should use the `manned.org` page.

Reply:
Agreed. For a record, I did not know about manned.org, I compared arch to linux.die.net, found out that both versions differ, assumed arch is more relevant, pasted address.